go/example_code/s3/s3.go

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
I added some waiters for create/delete bucket and committed to master, so I'm removing this file from the PR. I'll do the same for the Ruby example.